crates/cheatnet/src/execution/syscalls.rs

```diff
@@ -297,6 +298,17 @@ impl CheatableSyscallHandler<'_> {
             // Increment, since the selector was peeked into before
             self.syscall_handler.syscall_ptr += 1;
             return self.execute_syscall(vm, deploy_syscall, constants::DEPLOY_GAS_COST);
+        } else if SyscallSelector::EmitEvent == selector {
+            let result = self.syscall_handler.execute_next_syscall(vm, hint);
+            let event = Event::from_ordered_event(
+                self.syscall_handler.events.last().unwrap(),
```

Comment:
Are we sure there will be only one event? (Or if we want to catch only the last one)

Reply:
It's a good thing to check, but i doubt that they are batched

Reply:
Added an assert